course-info.yaml

```diff
@@ -25,17 +25,24 @@ vendor:
   url: https://www.jetbrains.com/
 programming_language: Rust
 content:
-- Introduction
-- Common Programming Concepts
-- Understanding Ownership
-- "Structs, Methods, Enums, and Pattern Matching"
-- Modules
-- Common Collections
-- Error Handling
-- "Generic Types, Traits, and Lifetime"
-- Writing Automated Tests
-- Standard Library Types
-- Fearless Concurrency
-- Macros
+  - Introduction
+  - Common Programming Concepts
+  - Understanding Ownership
+  - "Structs, Methods, Enums, and Pattern Matching"
+  - Modules
+  - Common Collections
+  - Error Handling
+  - "Generic Types, Traits, and Lifetime"
```

Comment:
do we need the double quotes?

Reply:
Added automatically by the plugin. I guess because of commas.